app/controllers/questions_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
   end
 
   def show
+    @question.increment_views
```

Comment:
Instead of redefining the `increment_views` on the questions model. You can use the ActiveRecord `increment!` method, like so: `@question.increment!(:views, 1)`